packages/block-library/src/column/edit.native.js

```diff
@@ -111,30 +111,22 @@ function ColumnEdit( {
 		onChangeWidth( nextWidth );
 	};
 
-	const renderAppender = useCallback( () => {
-		const { width: columnWidth } = contentStyle[ clientId ];
-		const isFullWidth = columnWidth === screenWidth;
-
-		if ( isSelected ) {
-			return (
-				<View
-					style={ [
-						styles.columnAppender,
-						isFullWidth && styles.fullwidthColumnAppender,
-						isFullWidth &&
-							hasChildren &&
-							styles.fullwidthHasInnerColumnAppender,
-						! isFullWidth &&
-							hasChildren &&
-							styles.hasInnerColumnAppender,
-					] }
-				>
-					<InnerBlocks.ButtonBlockAppender />
-				</View>
-			);
-		}
-		return null;
-	}, [ contentStyle[ clientId ], screenWidth, isSelected, hasChildren ] );
+	const { width: columnWidth } = contentStyle[ clientId ];
+	const isFullWidth = columnWidth === screenWidth;
+	const appender = isSelected && (
```

Comment:
I think this is not the same as the original? We should be explicitly return `null` if `isSelected` is falsy?

Reply:
It would work because the implementation is a bit different in the RN component (rendering the falsy values except the `undefined`), but yeah, makes sense to use `null` to follow the documentation and the same contract as the non-RN implementation. =)
Good catch! Also fixed and amended in the same commit: https://github.com/WordPress/gutenberg/pull/29911/commits/463ab85a5ca460d5e8fbd58dbe4bbd9b0a59ea8e

Reply:
Thinking more, I refactored the behavior on the non-RN component to keep the same behavior for both components.

So as opposite I [commented previously](https://github.com/WordPress/gutenberg/pull/29911#discussion_r664890605), not when the appender contains any value different of `undefined` that is falsy, it will return null for both cases.

https://github.com/WordPress/gutenberg/pull/29911/commits/70235deb80b80ac0b8cfce57c5abf16b4a2e5d2e

WDYT? In the documentation, I think we can keep only the `null` as recommended value to not render anything.

Reply:
Sure :). Sounds good to me.

Reply:
Just to let you know that it was changed again here: https://github.com/WordPress/gutenberg/pull/29911#discussion_r669365482 It seems to make more sense now =)